cpp/include/cudf/dictionary/dictionary_column_view.hpp

```diff
@@ -0,0 +1,71 @@
+/*
+ * Copyright (c) 2020, NVIDIA CORPORATION.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+#pragma once
+
+#include <cudf/column/column_view.hpp>
+#include <cudf/column/column.hpp>
+
+namespace cudf
+{
+
+/**
+ * @brief A wrapper class for operations on a dictionary column.
+ *
+ * A dictionary column contains a set of keys and a column of indices.
+ * The keys are a sorted set of unique values for the column.
+ * The indices values are position indices into the keys.
```

Comment:
```suggestion
 * The indices represent the corresponding positions of each element's value in the keys.
```

Reply:
Please don't resolve without action. If you reject the suggestion, please explain why.